.devcontainer/avatax/devcontainer.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "AvaTax app",
+  "dockerComposeFile": "docker-compose.yml",
+  "service": "avatax",
+  "workspaceFolder": "/app",
+  "forwardPorts": [3000, "dynamodb:8000"],
```

Comment:
question: how does it work - is it necessary? If dynamoDB is inside container, we need to expose it? or is it something for toolbox usage (if yes, probably we can add some documentaton)